sustainability-best-practices/frontend/README.md

```diff
@@ -0,0 +1,32 @@
+---
+description: >-
+  Here you can learn about how to optimize assets, implement a mobile-first strategy and other suggestions.
+---
+
+# Overview
+
+In this section, you can find some tips to optimize your website's performance and reduce its carbon footprint:
+
+## Page Weight Budget
+
+Choose how to transfer data for web page access minimizing the impact on carbon emissions.
+
+## Image Optimization
+
+Learn how to optimize images before uploading to a web page and usage of different file formats.
```

Comment:
it is not really a step by step guide but more like suggestions on what tools one can use etc
```suggestion
Optimize images before uploading to a web page and make use of different file formats.
```